exporter/clickhouselogsexporter/exporter.go

```diff
@@ -41,15 +42,19 @@ import (
 )
 
 const (
-	DISTRIBUTED_LOGS_TABLE     = "distributed_logs"
-	DISTRIBUTED_TAG_ATTRIBUTES = "distributed_tag_attributes"
+	DISTRIBUTED_LOGS_TABLE                      = "distributed_logs"
+	DISTRIBUTED_TAG_ATTRIBUTES                  = "distributed_tag_attributes"
+	DISTRIBUTED_LOGS_TABLE_V2                   = "distributed_logs_v2"
+	DISTRIBUTED_LOGS_RESOURCE_BUCKET_V2         = "distributed_logs_v2_resource_bucket"
```

Comment:
sure, let me get review from others will make this change at end as it will require migration for users who are currently using image built from this